app/assets/javascripts/discourse/app/lib/render-tag.js

```diff
@@ -44,6 +44,7 @@ export function defaultRenderTag(tag, params) {
     href +
     " data-tag-name=" +
     tag +
+    (params.description ? ' title="' + params.description + '" ' : "") +
```

Comment:
This description is null when showing the tags at /tags, could we add it there? It does not seem to flow through render-tags:

![image](https://user-images.githubusercontent.com/920448/143964011-54b2400b-21bb-4c33-9cbd-3fbbcca82782.png)

Reply:
fixed :)